src/main/charts/confluence/values.yaml

```diff
@@ -375,6 +375,11 @@ volumes:
   # those needs to be defined in synchrony.additionalVolumeMounts
   #
   additionalSynchrony: []
+  
+  # -- Mode bits used to set permissions on created files by default. Must be an octal value between 0000 and 0777 or a decimal value between 0 and 511
+  # Typically overriden in volumes from Secrets and ConfigMaps to make mounted files executable
+  #
+  defaultMode: 484
```

Comment:
Minor - would it be clearer if this was renamed to `defaultPermissionsMode`

Reply:
I used the name as it appears in K8S API assuming that users who are going to override it know exactly what they are doing. Renaming it should not be a big deal, I just wanted to stick to kube api names.

Reply:
Renamed